opm/parser/share/keywords/000_Eclipse100/C/COMPDAT

```diff
@@ -6,7 +6,7 @@
     {"name" : "K2" , "value_type" : "INT" },
     {"name" : "STATE" , "value_type" : "STRING" , "default" : "OPEN"},
     {"name" : "SAT_TABLE" , "value_type" : "INT" , "default" : 0},
-    {"name" : "CONNECTION_TRANSMISSIBILITY_FACTOR"     , "value_type" : "DOUBLE", "dimension" : "Viscosity*LiquidVolume/Time*Pressure"},
+    {"name" : "CONNECTION_TRANSMISSIBILITY_FACTOR"     , "value_type" : "DOUBLE", "dimension" : "Viscosity*ReservoirVolume/Time*Pressure"},
```

Comment:
Just an off question here: how do you parse this expression? Is there a parser inside the opm-parser...? To me the expression looks indefinite, unless one assumes left associativity, in which case it is wrong. So I just do not understand how this can end up correct! (Apparently, it does, or have we not tested FIELD cases with given factors?)